### PR TITLE
Adding a 'maxAge' flag, so we can tell the browser to cache assets.

### PR DESCRIPTION
**Pull request by evert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,13 @@ Getting started
...
 import { Application } from '@curveball/core';
 import serveFiles from '@curveball/static';
+import { join } from 'path';
 
 const app = new Application();
 app.use(
     serveFiles({
         // Absolute path to assets directory
-        staticDir: `${process.cwd()}/assets`,
+        staticDir: join(__dirname, 'assets'),
```
Comment (mhum) [inline README.md:24]: ++ Ah, right. This is probably a more predictable file path

File: test/serve.ts [hunk 2]
```diff
@@ -71,4 +71,24 @@ describe('serve', () => {
...
+        staticDir: `${process.cwd()}/test/assets`,
+        maxAge: 3600
+      }),
+      ctx
+    );
+
+    expect(ctx.response.body instanceof Buffer).to.be.true;
+    expect(ctx.response.body.toString()).to.equal('test static file');
+    expect(ctx.response.type).to.equal('text/plain');
+    expect(ctx.response.headers.get('Cache-Control')).to.equal('max-age=3600');
```
Comment (mhum) [inline test/serve.ts:93]: Should also verify the header _isn't_ returned if the option isn't provided.

File: changelog.md [hunk 3]
```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+0.1.1 (????-??-??)
```
Comment (mhum) [inline changelog.md]: Should this just be released as 1.0? It's hard to determine if this is a feature addition or bug fix using sub-1.0.0 version strings.

**Review by mhum — APPROVED** [review]
Looks good! Just a couple of small comments. You can make the changes or not; I don't feel strongly about them.

File: changelog.md (hunk 3 above)
Comment (evert) [reply changelog.md]: So far all the curveball packages start with a `0.`. I was waiting with 1 everywhere until, idk.. more stuff is done.. like a complete website .. 

But i can do 0.2 for this =)

**Pull request merged**